src/styles/forms/_inputs.scss

```diff
@@ -5,25 +5,25 @@
   position: relative;
 
   .hovered:not(.readonly) {
-    color: var(--KInputHover, var(--blue-500));
+    color: var(--KInputHover, var(--blue-500, #1155cb));
     transition: color 0.1s ease;
   }
 
   .focused:not(.readonly) {
-    color: var(--KInputFocus, var(--blue-500));
+    color: var(--KInputFocus, var(--blue-500, #1155cb));
     transition: color 0.1s ease;
   }
 
   .is-required {
-    color: var(--KLabelRequiredAsteriskColor, var(--KInputBorder, var(--grey-600)));
+    color: var(--KLabelRequiredAsteriskColor, var(--KInputBorder, var(--grey-600, #3c4557)));
     font-size: 11px;
     font-weight: 500;
     margin-left: var(--spacing-xxs);
```

Comment:
Can you ensure all of the old custom properties have a fallback value?

Reply:
Went through them all to make sure they have fallback values